syntax = "proto3";

package whisper;

service WhisperService {
  rpc VoiceToText (WisperRequest) returns (WisperResponse);
}

message WisperRequest {
  string url = 1;
  string openai_api_key = 2;
}


message WisperResponse {
  string answer = 1;
}